react/BottomSheet/BottomSheet.jsx

```diff
@@ -120,7 +120,10 @@ const BottomSheet = ({ toolbarProps, settings, children }) => {
   }
 
   const overriddenChildren = React.Children.map(children, child => {
-    if (child.type.name === 'BottomSheetHeader') {
+    if (
+      child.type.name === 'BottomSheetHeader' ||
+      child.type.displayName === 'BottomSheetHeader'
+    ) {
```

Comment:
I don't understand why the Argos snapshot (of BottomSheet) looks broken

Reply:
c'est comme le viewer, souci d'animation...